project-fortis-services/src/resolvers/shared.js

```diff
@@ -247,6 +252,7 @@ function withCsvExporter(promiseFunc, exportPropertyName, container, expiryMinut
 }
 
 module.exports = {
+  isCurrentUser,
```

Comment:
I'd move this function to where it's being used (and import the getUserFromArgs method from here) since I don't see many contexts outside of the user editing in which this method would be re-used.